apps/public-docsite-v9/src/Concepts/Upgrade/FromV8/Components/Input.stories.mdx

```diff
@@ -145,7 +145,7 @@ This table maps v8 `TextField` props to the v9 `Input` equivalent.
 | `onChange`                 | `onChange`                                 | Typescript types have changed                                                        |
 | `onNotifyValidationResult` | n/a                                        | v9 `Input` does not handle validation                                                |
 | `onGetErrorMessage`        | n/a                                        | v9 `Input` does not handle error states                                              |
-| `deferredValidationTime`   | n/a                                        | v9 `Input` does not handle validation                                                  |
+| `deferredValidationTime`   | n/a                                        | v9 `Input` does not handle validation                                                |
```

Comment:
What changed here? Whitespace?

Reply:
NVM, I can see the whitespace in the conversation tab. It wasn't visible in the review tab.